src/metabase/api/card.clj

```diff
@@ -208,25 +208,38 @@
             valid-metadata?)
        ;; only sent valid metadata in the edit. Metadata might be the same, might be different. We save in either case
        (and (nil? query)
+            valid-metadata?)
+
+       ;; copying card and reusing existing metadata
+       (and (nil? original-query)
+            query
             valid-metadata?))
-      (a/to-chan! [metadata])
+      (do
+        (log/debug (trs "Reusing provided metadata"))
```

Comment:
do we wish to keep these `log/debug`? Just double-checking because it uses i18n and I think we don't want too many translations for things we don't really need.

Reply:
I see no downside to having the string. And when you want this information it is quite nice to have.